Makefile.in

```diff
@@ -172,11 +175,28 @@ uninstall:
 	@rm -rf -v --preserve-root=all $(DESTDIR)$(DATADIR)/$(INSTALLDIR)
```

Comment:
I think this part of the `uninstall` rule is not relevant any more, as it deletes `$PREFIX/share/umu-launcher`, which I don't think we use since distributing as a python package. Should it be removed?